### PR TITLE
Fix broken links to API pages and overview

### PR DESCRIPTION
**Pull request by BrianTJackett**
Links incorrectly pointing to Markdown files as well as incorrect location for Export Teams content page.

### CODE REVIEW

**Review by MariaWissler — APPROVED** [review]

File: concepts/teams-licenses.md [hunk 1]
```diff
@@ -12,9 +12,9 @@ This article describes the licensing and payment requirements for the Microsoft
 
 Some APIs provide the option to choose a licensing and payment model via the `model` query parameter; others only support one model or do not support a licensing and payment model. The following APIs have consumption charges:
 
-* [Export Teams content](/graph/api/export-teams-content.md)
-* [Create subscription](/graph/api/subscription-post-subscriptions.md)
-* [Update chat message](/graph/api/chatmessage-update.md)
+* [Export Teams content](/microsoftteams/export-teams-content)
```
Comment (MariaWissler) [inline concepts/teams-licenses.md:15]: @BrianTJackett this link stills broken, thank you so much for pointing this before

Comment (BrianTJackett) [reply concepts/teams-licenses.md:15]: The full URL would be "https://docs.microsoft.com/microsoftteams/export-teams-content".  I was hoping to use a relative path but that may not work as it is under Microsoft Teams docs and not Graph.

**Comment by MariaWissler** [comment]
We need help merging this since the live links are currently broken

File: concepts/teams-licenses.md (hunk 1 above)
Comment (Lauragra) [reply concepts/teams-licenses.md:15]: The relative path should work it just won't work if you're testing it in your branch.

**Pull request merged**